.travis.yml

```diff
@@ -5,12 +5,25 @@ env:
   - secure: 'CicwL3ulhGLS+J8yvSQt18Ta1VUOBKv9Rd60ZMzlkCoEXAoO7IvpYUwnUXE7ZurLRt1Yuif+wd3D7P4BNDFIOVw7gEYllX9S4URE+9j3I/oFze+dFpqncb15CWkCK6EIyerXs/HjZe/J1o0IwPmDTqn6KXBnP6wCNLBf4UoJBnw='
 sudo: false
 language: python
-python:
-- '3.6'
-- '3.7'
-- '3.8'
-install: pip install tox-travis
-script: tox
+jobs:
+  include:
+  - python: '3.6'
+    env: TOXENV=docs
+  - python: '3.6'
+    env: TOXENV=lint
+  - python: '3.6'
+    env: TOXENV=clean,py36,stats
```

Comment:
is the clean step here necessary still?

Reply:
ah, yea, its to blow away coverage artifacts if present. It seems common in other travis integration libraries, so I assume its doing something but haven't validated